### PR TITLE
Beginning of Build.scala cleanup!

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

File: kifi-backend/project/Build.scala [hunk 1]
```diff
@@ -222,15 +222,10 @@ object ApplicationBuild extends Build {
     EclipseKeys.skipParents in ThisBuild := false,
     sources in doc in Compile := List(),
     Keys.fork := false,
-    // Keys.fork in Test := false, // uncomment to hook debugger while running tests
```
Comment (martinraison) [inline kifi-backend/project/Build.scala:225]: why are you removing this? I used it very often

Comment (andrewconner) [inline kifi-backend/project/Build.scala:225]: I can add it back — trying to remove dead code that makes it more confusing.

**Comment by 2is10** [comment]
10000 w00t

**Comment by raymondng** [comment]
:+1:

File: kifi-backend/project/Build.scala (hunk 1 above)
Comment (martinraison) [inline kifi-backend/project/Build.scala:225]: sure, I guess if nobody else was using it it doesn't really matters.

**Pull request merged**